docs/07_admin/01_REST.rst

```diff
@@ -18,5 +18,273 @@
 The FogLAMP REST API
 ********************
 
-.. note:: This page is currently under construction. Come back soon to check it again!
+Users, admnistrators and applications interact with FogLAMP via a REST API. This section presents a full reference of the API.
+
+.. note:: The FogLAMP REST API should not be confused with the internal REST API used by FogLAMP tasks and microservices to communicate with each other.
+
+
+Introducing the FogLAMP REST API
+================================
+
+The REST API is the route into the FogLAMP appliance, it provides all user and program interaction to configure, monitor and manage the FogLAMP system. A separate specification will define the contents of the API, in summary however it is designed to allow for: 
+
+- The complete configuration of the FogLAMP appliance
+- Access to monitoring statistics for the FogLAMP appliance
+- User and role management for access to the API
+- Access to the data buffer contents
+
+
+Port Usage
+----------
+
+In general FogLAMP components use dynamic port allocation to determine which port to use, the admin API is however an exception to this rule. The Admin API port has to be known to end-users and any user interface or management system that uses it, therefore the port on which the admin API listens must be consistent and fixed between invocations. This does not mean however that it can not be changed by the user. The user must have the option to define the port to use by the admin API to listen on. To achieve this the port will be stored in the configuration data for the admin API, using the configuration category *AdminAPI*, see Configuration. Administrators who have access to the appliance can find information regarding the port and the protocol to used (i.e. HTTP or HTTPS) in the *pid* file stored in *$FOGLAMP_DATA/var/run/*:
```

Comment:
The category in the code currently is called rest_api - not sure which we said we wanted, I suspect it was what is documented here.